.github/workflows/quest.yml

```diff
@@ -51,7 +52,7 @@ jobs:
         uses: dotnet/docs-tools/actions/sequester@main
         env:
           ImportOptions__ApiKeys__GitHubToken: ${{ secrets.GITHUB_TOKEN }}
-          ImportOptions__ApiKeys__AzureAccessToken: ${{ env.AZURE_ACCESS_TOKEN }}
+          ImportOptions__ApiKeys__AzureAccessToken: ${{ steps.azure-oidc-auth.outputs.access-token }}
           ImportOptions__ApiKeys__QuestKey: ${{ secrets.QUEST_KEY }}
           ImportOptions__ApiKeys__SequesterPrivateKey: ${{ secrets.SEQUESTER_PRIVATEKEY }}
           ImportOptions__ApiKeys__SequesterAppID: ${{ secrets.SEQUESTER_APPID }}
```

Comment:
Same comment here.  Need to add 

```
duration: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.duration || 5 }}
```
As in https://github.com/dotnet/docs/blob/main/.github/workflows/quest-bulk.yml#L51C11-L51C105

Reply:
In dotnet/docs, this line is only in Quest-bulk.yml,  not Quest.yml -- should it only be in Quest-bulk.yml?